plugins/commands/watch.py

```diff
@@ -107,21 +110,60 @@ def docker(command):
     return subprocess.run(prefix + ["docker"] + command)
 
 
+class WatchingBuilder(object):
```

Comment:
Move this into a common library so that it can be shared and used by other commands and separate out the `watch` and `serve` commands.

Reply:
Done.